frontend/model/notifications/templates.js

```diff
@@ -43,10 +46,23 @@ export default ({
       errMsg: error.message ?? '?'
     }
 
+    const LplaintextParams = {
+      errName: error.name,
+      activity,
+      action: action ?? opType,
+      contract: state.contracts[contractID]?.type ?? contractID,
+      who: plaintextWho,
+      errMsg: error.message ?? '?'
+    }
```

Comment:
A lot of repetition here between `Lparams` and `LplaintextParams`, can it be DRY'd please using the spread operator?